storage/src/main/java/org/apache/kafka/storage/internals/log/RemoteStorageThreadPool.java

```diff
@@ -34,7 +34,7 @@
 import static org.apache.kafka.server.log.remote.storage.RemoteStorageMetrics.REMOTE_STORAGE_THREAD_POOL_METRICS;
 
 public class RemoteStorageThreadPool extends ThreadPoolExecutor {
-    private final Logger logger;
+    private final Logger logger = LoggerFactory.getLogger(RemoteStorageThreadPool.class);
```

Comment:
Can you make this as static to avoid creating multiple logger instances as we do not need any context for each instance?

Reply:
> I understand that we create only a single instance for now but it is good to make this logger static even if we create multiple instances in future.

that is a good point. The default provider `reload4j` maintain the logger instance for each class name, so the instance should be same even though we don't use `static`. However, it seems slf4j docs does not describe such behavior so we can't assume other providers are same

for another, we should apply the `static` to code base. 1) unify the code style and 2) avoid multiple logger instances in the future.